packages/eslint-config-eslint/package.json

```diff
@@ -3,15 +3,22 @@
   "version": "8.0.0",
   "author": "Nicholas C. Zakas <nicholas+npm@nczconsulting.com>",
   "description": "Default ESLint configuration for ESLint projects.",
+  "exports": {
+    "./package.json": "./package.json",
+    ".": "./index.js",
+    "./cjs": "./cjs.js"
+  },
```

Comment:
I didn't see a need to add this - we did this in the eslint package to avoid other devs using the internal modules, but this package is (mostly) only used by eslint. :)

Reply:
I thought it would be good to clarify which modules are intended to be public.

And with this, `import "eslint-config-eslint/cjs"` doesn't need `.js` extension :)

Reply:
we need to export `./eslintrc` as well - it was used in `.eslintrc.js`: https://github.com/eslint/eslint/blob/cdd063c388bbfe1781d7a864a832f03a2c1cc277/.eslintrc.js#L65

Reply:
Good catch! Updated in https://github.com/eslint/eslint/pull/17336/commits/6f71c8efe1f66aff9ad43eaf135bd33b1e8f4be8